draft-thomson-webpush-protocol.xml

```diff
@@ -315,16 +303,20 @@ Host: push.example.net
 ]]></artwork>
       </figure>
       <t>
-        A response with a 201 (Created) status code includes a URI for the
-        "subscription" resource in the Location header field.
+        A response with a 201 (Created) status code MUST include a URI for the
+        new "subscription" resource in the Location header field.
+      </t>
+      <t>
+        The push server MUST also return new "push" and "receipt subscribe"
```

Comment:
Perhaps instead:
> The push server MUST provide a URI for a new push resource that corresponds to this subscription in a link relation of type "....:push".   The push server MUST provide a URI for a receipt subscribe resource in a link relation of type "...:r..:s..".  

Note that the receipt subscription resource might be reused.  Actually, it should be, so that application servers can detect where existing receipt subscriptions can be used.

It's probably worthwhile noting that point.